modules/apps/content-dashboard/content-dashboard-web/src/main/java/com/liferay/content/dashboard/web/internal/portlet/action/GetContentDashboardItemInfoMVCResourceCommand.java

```diff
@@ -209,6 +233,51 @@ private JSONArray _getAssetTagsJSONArray(
 			).toArray());
 	}
 
+	private JSONObject _getAssetVocabulariesJSONObject(
+		ContentDashboardItem contentDashboardItem, Locale locale) {
+
+		List<AssetCategory> assetCategories =
+			contentDashboardItem.getAssetCategories();
+
+		Stream<AssetCategory> stream = assetCategories.stream();
+
+		HashMap<Long, Map<String, Object>> assetVocabularies = new HashMap<>();
+
+		stream.forEach(
+			assetCategory -> {
+				long vocabularyId = assetCategory.getVocabularyId();
+
+				if (Validator.isNull(assetVocabularies.get(vocabularyId))) {
```

Comment:
Better if we use Optional interface instead https://www.baeldung.com/java-optional